core/src/main/java/io/snyk/plugins/artifactory/SnykPlugin.java

```diff
@@ -81,32 +82,32 @@ public void handleBeforeDownloadEvent(RepoPath repoPath) {
     try {
       scannerModule.scanArtifact(repoPath);
     } catch (CannotScanException e) {
-      LOG.debug("Artifact cannot be scanned {}. {}", repoPath, e.getMessage());
+      LOG.debug("Artifact cannot be scanned. {} {}", e.getMessage(), repoPath);
     } catch (SnykAPIFailureException e) {
       final String blockOnApiFailurePropertyKey = SCANNER_BLOCK_ON_API_FAILURE.propertyKey();
       final String blockOnApiFailure = configurationModule.getPropertyOrDefault(SCANNER_BLOCK_ON_API_FAILURE);
       final String causeMessage = Optional.ofNullable(e.getCause())
         .map(Throwable::getMessage)
-        .map(m -> " " + m)
-        .orElse("");
+        .map(m -> e.getMessage() + " " + m)
+        .orElseGet(e::getMessage);
 
-      String message = "Failed to scan artifact '" + repoPath + "'. " + e.getMessage() + causeMessage;
+      String message = format("Artifact scan failed. %s %s", causeMessage, repoPath);
       if ("true".equals(blockOnApiFailure)) {
         throw new CancelException(message, 500);
       }
       LOG.debug(message);
-      LOG.debug("Property '{}' is false, so allowing download: '{}'", blockOnApiFailurePropertyKey, repoPath);
+      LOG.debug("Allowing download. Plugin Property \"{}\" is not \"true\". {}", blockOnApiFailurePropertyKey, repoPath);
```

Comment:
I used "is not true" instead of "is false" because we don't valid booleans currently, we just take "true" and anything else is false.